pyproject.toml

```diff
@@ -165,6 +165,7 @@ disable = [
 [tool.pytest.ini_options]
 testpaths = ["tests", "src"]
 asyncio_mode = "auto"
+asyncio_default_fixture_loop_scope = "function"
```

Comment:
Maybe we should remove this and keep the default as it is?

Reply:
Then we get a depreciation warning saying the behavior will change and one should set a default

Reply:
Oh, OK, I'm confused now. So this has nothing to do with trying to make tests pass and is just a new deprecation message in a new version of `pytest-asyncio`?

Reply:
I def. tried this to make the test pass, bit it also fixes a deprecation warning, it just didn't help though.